common/output-model.cpp

```diff
@@ -991,7 +991,9 @@ void output_model::run_command(std::string command, device_models_list & device_
                     found = true;
                     auto res = dbg.send_and_receive_raw_data(buffer);
 
-                    std::string response = rsutils::string::from() << "\n" << terminal_parser.parse_response(to_lower(command), res);
+                    int opcode = *res.data() - 256; // error opcodes are in negative numbers
```

Comment:
IMO this pointer access should be protected

Reply:
How should we use `std::abs()`? `int opcode = - std::abs(*res.data() - 256);` ?
we want the value of `opcode` to be negative

Reply:
Don't we already get the value as negative?
If so I aimed for
`int opcode = - std::abs(*res.data());`

Reply:
`int8_t opcode = std::abs(*res.data());`